### DISCUSSION TITLE
Can't apply for student pack

### DISCUSSION THREAD

**Discussion by babyharpsea** [community/community#32782] [GitHub Education]
I've tried following everything it said, and now the only thing that's popping up is:

`Your application cannot be reviewed until you fix the following:

You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.`

I've already put in my user the info requiered by GitHub as said in the documentation link.. what else do I need to do? Please help..

**Comment by babyharpsea** [comment]
Hello, anyone? I could apply directly before, but now I'm not able to...

**Comment by diamant3** [comment]
Hi! @babyharpsea try this [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#applying-for-a-github-student-developer-pack), but if you tried and your problem is not solved, then submit a ticket [here](https://support.github.com)
